base/applications/mscutils/eventvwr/lang/de-DE.rc

```diff
@@ -217,14 +220,15 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_COPY "Typ:            %s\r\n\
-Quelle:         %s\r\n\
-Kategorie:      %s\r\n\
-Ereignis-ID:    %s\r\n\
-Datum:          %s\r\n\
-Zeit:           %s\r\n\
-Benutzer:       %s\r\n\
-Computer:       %s\r\n\
+    IDS_COPY "\
+Typ:         %s\r\n\
+Quelle:      %s\r\n\
+Kategorie:   %s\r\n\
+Ereignis-ID: %s\r\n\
+Datum:       %s\r\n\
+Zeit:        %s\r\n\
+Benutzer:    %s\r\n\
+Computer:    %s\r\n\
```

Comment:
The purpose of the "extra" spaces was to have the values (`%s\r\n`) aligned on tab-length boundaries, since this is copiable data (that gets stored in the registry) for the user..
So please revert this change (here and in the other files where this has been done)

Reply:
Personally I don't see the value yet in aligning that to a multiple of 4 spaces, although I am/was aware of its usage for copying data out. To me aligning it at the rightmost ':' makes more sense, but I can restore the multiple-of-4-spaces if you consider that to be an absolute must.

Reply:
I followed your suggestion of restoring 4-space-indentation, but as you can see this still leads to changes in some of the languages, actually proving your claim that we would use 4-space-indentation here everywhere by intention to be wrong. I feel the result being very aesthetically unappealing now, and perceived the state of rightmost `: %s\r\n` that I used before vastly superior.

Reply:
Why does it have to be a multiple of 4? Can't we just use one space for all langs, and 2 spaces for french maybe as the outlier?

Reply:
> all of them changed now by following your suggestion == proving that they did not follow your suggestion beforehand

Because these have been translated _**after**_ the english and french translations have been made and nobody in those other languages respected the alignment thing.

Reply:
Yes, and I don't feel obliged to that artificial obligation neither, same as the other translators. Note, that I didn't transform anything away from the 4 spaces indentation in this PR! To respect your feelings. Everything I touched in that padding regard now with the final commit turned some which were `spaces module 4 != 0` before and became `spaces modulo 4 == 0` after. But like I said: I didn't turn any of them to that which would have required me to use the space key. I do hope that we can both live with that compromise for master. If you want to enforce the space-bloat everywhere, you will have to do that yourself.